azurerm/resource_arm_virtual_hub.go

```diff
@@ -71,6 +71,7 @@ func resourceArmVirtualHub() *schema.Resource {
 				ValidateFunc: azure.ValidateResourceID,
 			},
 
+			// TODO: these need to be managed via a separate resource
 			"virtual_network_connection": {
```

Comment:
You mean I need to add new data resource file and resource file for this field as a separate resource?

Reply:
I've got a PR coming adding these - but yes, these need to be managed as separate resources